src/style.css

```diff
@@ -1,3 +1,52 @@
 body {
   margin: 0;
+  display: block;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
```

Comment:
You don`t need you set flex-direction: row because this is default direction of flex box